### DISCUSSION TITLE
Displaying Daily Active Users for your time zone?

### DISCUSSION THREAD

**Discussion by sahandnayebaziz** [TelemetryDeck/Viewer#423] [Q&A]
Hi everyone, 

Does the dashboard support showing live DAU count for the current day? For example, if you're based in a particular time zone, will you see the DAU chart update as the day progresses?

I am based in the US in Pacific time and do not see the proper current day. For example, if it's July 18th, the dashboard will show a count for July 17th at the rightmost index of the chart for most of the day. I was hunting around for a Time Zone setting but I couldn’t find any. I assumed the dashboard is supposed to be “live” though right?

Thank you

**Comment by winsmith** [comment]
We currently don't support setting a time zone, but it is on our roadmap. Here's the ticket: https://github.com/TelemetryDeck/Issues/issues/203
